Makefile.am

```diff
@@ -147,30 +147,36 @@ indent:
 	clang-format -i *.c *.h test/*.c test/*.h
 
 EXTRA_DIST = \
+	autogen.sh \
 	CHANGES \
+	.clang-format \
+	contrib/95-radvd-gen \
+	contrib/unset_addrs.sh \
 	COPYRIGHT \
-	INTRO.html \
 	copyright.blurb \
+	.gitignore \
+	INTRO.html \
 	radvd.8.man \
 	radvd.conf.5.man \
 	radvd.conf.example \
 	radvd.service.in \
 	radvdump.8.man \
-	redhat/SysV/radvd-tmpfs.conf \
+	redhat/systemd/radvd.service \
+	redhat/systemd/radvd.spec \
+	redhat/systemd/radvd-tmpfs.conf \
 	redhat/SysV/radvd.conf.empty \
 	redhat/SysV/radvd.init \
 	redhat/SysV/radvd.spec \
 	redhat/SysV/radvd.sysconfig \
-	redhat/systemd/radvd-tmpfs.conf \
-	redhat/systemd/radvd.service \
-	redhat/systemd/radvd.spec \
+	redhat/SysV/radvd-tmpfs.conf \
 	test/check.c \
 	test/print_safe_buffer.c \
 	test/print_safe_buffer.h \
 	test/send.c \
 	test/test1.conf \
 	test/test_rdnss.conf \
-	test/util.c
+	test/util.c \
+	.travis.yml
```

Comment:
I'm not used to autotools, so my question may look stupid. Why is the repository CI file embedded in the distributed files ? It does not seem useful out of this repository for me.

Reply:
> It does not seem useful out of this repository for me.

I agree on that. And I admit I that my _looks good to me, LGTM_  was mostly for breaking the silence to this  merge request.

Reply:
Including these files makes it easier to specifically verify tarballs that are automatically generated from tags would otherwise match the `dist` tarballs as closely as possible. I agree it's NOT strictly required.

One of the driving factors behind this PR, was that I had to review the 2.19 release tarballs that Reuben generated, since I didn't generate them myself, I couldn't be 100% sure what was included in the configure script did not contain any hidden payloads. By including all files into the dist tarball, and making all autotools-generated files come from a reproducible container environment, I made it easy for both myself and anybody else to verify that what was included was a very close match to the git repo & automatic tags, and if they still didn't want to trust the `dist` tarballs, they could reproduce them via the autogen Docker container.